go.mod

```
@@ -1,10 +1,10 @@
-module github.com/ajeetdsouza/testcontainers-aerospike-go
+module github.com/bitcoin-sv/testcontainers-aerospike-go
 
-go 1.21
+go 1.24.0
 
 require (
-	github.com/aerospike/aerospike-client-go/v6 v6.13.0
-	github.com/stretchr/testify v1.9.0
+	github.com/aerospike/aerospike-client-go/v8 v8.2.0
+	github.com/stretchr/testify v1.10.0
 	github.com/testcontainers/testcontainers-go v0.31.0
```

suggestion: bump TC to the latest release too
```suggestion
	github.com/testcontainers/testcontainers-go v0.36.0
```